test/e2e/storage/utils/utils.go

```diff
@@ -191,14 +190,11 @@ func TestVolumeUnmountsFromDeletedPodWithForceOption(c clientset.Interface, f *f
 	}
 	Expect(err).NotTo(HaveOccurred())
 
-	// Wait for pod to enter "Terminating state"
-	time.Sleep(30 * time.Second)
-
 	By("Starting the kubelet and waiting for pod to delete.")
 	KubeletCommand(KStart, c, clientPod)
-	err = f.WaitForPodTerminated(clientPod.Name, "")
-	if !apierrs.IsNotFound(err) && err != nil {
-		Expect(err).NotTo(HaveOccurred(), "Expected pod to terminate.")
+	err = f.WaitForPodNotFound(clientPod.Name, framework.PodStartTimeout)
```

Comment:
maybe have a separate timeout for pod deletion. There is a TODO for PodStartTimeout to change it to 30 second.

Reply:
done